packages/system/capi-operator/charts/cluster-api-operator/templates/ipam.yaml

```
@@ -1,8 +1,26 @@
-  {{- $ipamName := $name }}
-  {{- $ipamVersion := get $ipam "version" }}
+{{- if .Values.ipam }}
+{{- $ipams := split ";" .Values.ipam }}
+{{- $ipamNamespace := "" }}
+{{- $ipamName := "" }}
+{{- $ipamVersion := "" }}
+{{- range $ipam := $ipams }}
+{{- $ipamArgs := split ":" $ipam }}
+{{- $ipamArgsLen := len $ipamArgs }}
+{{-  if eq $ipamArgsLen 3 }}
+  {{- $ipamNamespace = $ipamArgs._0 }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)